src/winforms/toga_winforms/widgets/textinput.py

```diff
@@ -35,8 +39,17 @@ def rehint(self):
         self.interface.intrinsic.height = self.native.PreferredSize.Height
 
     def set_on_change(self, handler):
-        self.native.TextChanged += self.on_text_change
+        self.native.TextChanged += self.winforms_onTextChanged
```

Comment:
Is this right? Shouldn't this be in the create() method, setting up the Winforms handler?